consensus/validator.go

```diff
@@ -301,16 +313,19 @@ func (consensus *Consensus) onCommitted(msg *msg_pb.Message) {
 	if !consensus.isRightBlockNumCheck(recvMsg) {
 		return
 	}
+	if len(recvMsg.SenderPubkeys) != 1 {
```

Comment:
why not directly use isSendByLeader()

Reply:
Since currently there is no guarantee that the leader message is signed by leader key (on Testnet). The consensus will not work if we do check the leader public key. This will be resolved in future PR.

Reply:
The function key check is already outside of `onPrepared` and `onCommitted`. Thus we don't need to write this logic again. This piece of code is removed.